gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -37,6 +37,7 @@ public class HaGatewayConfiguration
     private List<String> extraWhitelistPaths = new ArrayList<>();
     private OAuth2GatewayCookieConfiguration oauth2GatewayCookieConfiguration = new OAuth2GatewayCookieConfiguration();
     private GatewayCookieConfiguration gatewayCookieConfiguration = new GatewayCookieConfiguration();
+    private List<String> customStatementPaths = new ArrayList<>();
```

Comment:
I think it would be better to have a statementPaths setup and it defaults to `v1/statement` .. and users can override or expand to multiple paths